src/app_service/service/devices/service.nim

```diff
@@ -89,6 +98,8 @@ QtObject:
 
     self.events.on(SignalType.LocalPairing.event) do(e:Args):
       let signalData = LocalPairingSignal(e)
+      if not signalData.accountData.isNil and signalData.accountData.keycardPairings.len > 0:
+        createKeycardPairingFile(signalData.accountData.keycardPairings)
```

Comment:
Could this be done in `status-go`? Or it knows nothing of `status-keycard-go`?

Reply:
Exactly, I was thinking about this and was like 50:50 to do it here or there and the only reason why I left it here is cause status-go doesn't know anything about keycard lib (status-keycard-go). 

To be clear, it's also possible to do this on the status-go side, but then we need to pass the `pairings.json` file location to status-go, cause client defines it and set to keycard lib and I see that like mixing contexts and opted in for this solution.